--- templates/edit_file_gallery.tpl	2010-10-25 10:03:31.000000000 -0200
+++ templates/edit_file_gallery.tpl	2010-11-23 11:04:07.590285000 -0200
@@ -27,8 +27,6 @@
 									<input type="hidden" name="name" value="{$gal_info.name|escape}" />
 								{else}
 									<input type="text" size="50" id="name" name="name" value="{$gal_info.name|escape}" style="width:100%"/>
-									<br/>
-									<em>{tr}Required for podcasts{/tr}.</em>
 								{/if}
 							</td>
 						</tr>
@@ -42,9 +40,8 @@
 									<input type="hidden" name="fgal_type" value="system" />
 								{else}
 									<select name="fgal_type" id="fgal_type">
-										<option value="default" {if $gal_info.type eq 'default'}selected="selected"{/if}>{tr}Any file{/tr}</option>
-										<option value="podcast" {if $gal_info.type eq 'podcast'}selected="selected"{/if}>{tr}Podcast (audio){/tr}</option>
-										<option value="vidcast" {if $gal_info.type eq 'vidcast'}selected="selected"{/if}>{tr}Podcast (video){/tr}</option>
+										<option value="default" {if $gal_info.type eq 'default'}selected="selected"{/if}>Acervo de arquivos</option>
+										<option value="images" {if $gal_info.type eq 'images'}selected="selected"{/if}>Galeria de imagens</option>
 									</select>
 								{/if}
 							</td>
@@ -55,8 +52,6 @@
 							</td>
 							<td>
 								<textarea rows="5" cols="40" id="description" name="description" style="width:100%">{$gal_info.description|escape}</textarea>
-								<br/>
-								<em>{tr}Required for podcasts{/tr}.</em>
 							</td>
 						</tr>
 						<tr>
