--- templates/tiki-upload_file_progress.tpl 2010-09-21 14:26:01.000000000 -0300
+++ templates/tiki-upload_file_progress.tpl	2010-11-23 12:27:28.990285002 -0200
@@ -26,7 +26,7 @@
 					{$feedback_message}
 				</div>
 			{/if}
-			{if empty($filegals_manager)}
+			{*{if empty($filegals_manager)}
 				<div>
 					{button href="#" _onclick="javascript:flip('uploadinfos$fileId');flip('close_uploadinfos$fileId','inline');return false;" _text="{tr}Additional Info{/tr}"}
 					<span id="close_uploadinfos{$fileId}" style="display:none">
@@ -77,7 +77,7 @@
 						</table>
 					{/if}
 				</div>
-			{/if}
+			{/if}*}
 		</td>
 	</tr>
 </table>
