--- templates/tiki-list_file_gallery.tpl	2010-03-18 11:48:32.000000000 -0300
+++ templates/tiki-list_file_gallery.tpl	2010-03-21 17:31:04.000000000 -0300
@@ -83,7 +83,7 @@
     {if $view eq 'browse'}
       {button _text="{tr}List Gallery{/tr}" href="?view=list&amp;galleryId=$galleryId"}
     {else}
-      {button _text="{tr}Browse Images{/tr}" href="?view=browse&amp;galleryId=$galleryId" _ajax='n'} {* no AJAX to make shadowbox work in browse view *}
+      {*{button _text="{tr}Browse Images{/tr}" href="?view=browse&amp;galleryId=$galleryId" _ajax='n'}*} {* no AJAX to make shadowbox work in browse view *}
     {/if}
   {/if}
 
@@ -191,9 +191,11 @@
   {include file='duplicate_file_gallery.tpl'}
 
 {else}
-{if $files or ($find ne '')}
+{*{if $files or ($find ne '')}
   {include file='find.tpl' find_show_num_rows = 'y'}
-{/if}
+{/if}*}
+<br /><br />
+
 {if $prefs.fgal_quota_show eq 'y' && $gal_info.quota}
 	<div style="float:right">
 		{capture name='use'}{math equation="round((100*x)/(1024*1024*y))" x=$gal_info.usedSize y=$gal_info.quota}{/capture}
