--- lib/tiki-js.js	2010-03-02 16:56:57.000000000 -0300
+++ lib/tiki-js.js	2010-03-21 22:43:43.000000000 -0300
@@ -1548,53 +1548,53 @@
    if (nScore >= 90)
    {
     var strIcon = "<img src='pics/icons/accept.png' style='vertical-align:middle' alt='Very Secure' />";
-    var strText = "Very Secure";
+    var strText = "Muito seguro";
     var strColor = "#0ca908";
    }
    // -- Secure
    else if (nScore >= 80)
    {
     strIcon = "<img src='pics/icons/accept.png' style='vertical-align:middle' alt='Secure' />";
-    strText = "Secure";
+    strText = "Seguro";
     vstrColor = "#0ca908";
   }
   // -- Very Strong
    else if (nScore >= 70)
    {
     strIcon = "<img src='pics/icons/accept.png' style='vertical-align:middle' alt='Very Strong' />";
-    strText = "Very Strong";
+    strText = "Muito forte";
     strColor = "#0ca908";
   }
   // -- Strong
    else if (nScore >= 60)
    {
     strIcon = "<img src='pics/icons/accept.png' style='vertical-align:middle' alt='Strong' />";
-    strText = "Strong";
+    strText = "Forte";
     strColor = "#0ca908";
   }
   // -- Average
    else if (nScore >= 40)
    {
     strIcon = " ";
-    strText = "Average";
+    strText = "Médio";
     strColor = "#e3cb00";
   }
   // -- Weak
    else if (nScore >= 25)
    {
     strIcon = "<img src='pics/icons/exclamation.png' style='vertical-align:middle' alt='Weak' />";
-    strText = "Weak";
+    strText = "Fraco";
     strColor = "#ff0000";
   }
   // -- Very Weak
    else
    {
     strIcon = "<img src='pics/icons/exclamation.png' style='vertical-align:middle' alt='Very weak' />";
-    strText = "Very Weak";
+    strText = "Muito fraco";
     strColor = "#ff0000";
   }
   ctlBar.style.backgroundColor = strColor;
-  ctlText.innerHTML = "<span>"  + strIcon + " Strength: " + strText + "</span>";
+  ctlText.innerHTML = "<span>"  + strIcon + " Força: " + strText + "</span>";
 }
 
 // Checks a string for a list of characters
@@ -1616,7 +1616,7 @@
 
 function checkPasswordsMatch(in1, in2, el) {
 	if ($jq(in1).val().length && $jq(in1).val() == $jq(in2).val()) {
-		$jq(el).html("<img src='pics/icons/accept.png' style='vertical-align:middle' alt='Secure' /><em>Passwords match</em>");
+		$jq(el).html("<img src='pics/icons/accept.png' style='vertical-align:middle' alt='Secure' /><em>Senhas coincidem</em>");
 		return true;
 	} else {
 		$jq(el).html("");
