--- templates/tiki-upload_file.tpl	2010-11-22 10:10:24.517205999 -0200
+++ templates/tiki-upload_file.tpl	2010-11-23 17:16:59.401047000 -0200
@@ -17,9 +17,9 @@
 	{if count($galleries) > 0 and $tiki_p_list_file_galleries eq 'y'}
 		{if $filegals_manager neq ''}
 			{assign var=fgmanager value=$filegals_manager|escape}
-			{button href="tiki-list_file_gallery.php?filegals_manager=$fgmanager" _text="{tr}List Galleries{/tr}"}
+			{button href="$base_url" _text="{tr}List Galleries{/tr}"}
 		{else}
-			{button href="tiki-list_file_gallery.php" _text="{tr}List Galleries{/tr}"}
+			{button href="$base_url" _text="{tr}List Galleries{/tr}"}
 		{/if}
 	{/if}
 	{if count($uploads) > 0}
@@ -113,7 +113,11 @@
 	{capture name=upload_file assign=upload_str}
 	<hr class="clear"/>
 
-
+    {if $gal_info.type eq 'images' and preg_match('/^image/', $fileInfo.filetype)}
+        <br />
+        <div align="center"><a href="{$base_url}tiki-download_file.php?fileId={$fileInfo.fileId}&display" target="_blank"><img src="{$base_url}tiki-download_file.php?fileId={$fileInfo.fileId}&display&x=300"></a></div>
+        <br />
+    {/if}
 
 	<div class="fgal_file">
 		<div class="fgal_file_c1">
@@ -124,13 +128,45 @@
 					<input style="width:100%" type="text" id="name" name="name[]" {if isset($fileInfo) and $fileInfo.name}value="{$fileInfo.name}"{/if} size="40" /> {if $gal_info.type eq "podcast" or $gal_info.type eq "vidcast"} ({tr}required field for podcasts{/tr}){/if}
 				</td>
 			</tr>
+			
+			{if isset($fileInfo) and $tiki_p_admin eq 'y'}
+				<tr>
+					<td><label for="name">Nome do arquivo:</label></td>
+					<td width="80%">
+						<input style="width:100%" type="text" id="filename" name="filename" {if isset($fileInfo) and $fileInfo.filename}value="{$fileInfo.filename}"{/if} size="40" />
+					</td>
+				</tr>
+				<tr>
+					<td><label for="name">Data de criação:</label></td>
+					<td width="80%">
+						{html_select_date prefix="created_" time=$fileInfo.created start_year="-5" end_year="+10" field_order=$prefs.display_field_order} {tr}at{/tr}
+				                <span dir="ltr">{html_select_time prefix="created_" time=$$fileInfo.created display_seconds=false}</span>
+					</td>
+				</tr>
+			{/if}
 			<tr>
 				<td><label for="description">{tr}File description:{/tr}</label></td>
 				<td>
 					<textarea style="width:100%" rows="2" cols="40" id="description" name="description[]">{if isset($fileInfo) and $fileInfo.description}{$fileInfo.description}{/if}</textarea>
 				{if $gal_info.type eq "podcast" or $gal_info.type eq "vidcast"}<br /><em>{tr}Required for podcasts{/tr}.</em>{/if}
 				</td>
-			</tr>{/if}
+			</tr>
+			{if isset($gal_info) and $gal_info.type eq 'images'}
+				<tr>
+					<td><label for="subtitle">Legenda:</label></td>
+					<td>
+						<textarea style="width:100%" rows="2" cols="40" id="subtitle" name="subtitle[]">{if isset($fileInfo) and $fileInfo.subtitle}{$fileInfo.subtitle}{/if}</textarea>
+					</td>
+				</tr>
+
+				<tr>
+					<td><label for="credit">Crédito:</label></td>
+					<td>
+						<input type="text" id="credit" name="credit[]" value="{if isset($fileInfo) and $fileInfo.credit}{$fileInfo.credit}{/if}" style="width:100%;" />
+					</td>
+				</tr>
+			{/if}
+			{/if}
 			<tr>
 	{* File replacement is only here when the javascript upload action is not
 	available in the file listing.
@@ -177,10 +213,9 @@
 				<input type="hidden" name="galleryId" value="{$prefs.fgal_root_id}"/>
 			{elseif empty($groupforalert)}
 			<tr><td>
-				<label for="galleryId">{tr}File gallery:{/tr}</label>
+				<label for="galleryId">Selecionar área:</label>
 			</td><td width="80%">
 				<select id="galleryId" name="galleryId[]">
-					<option value="{$prefs.fgal_root_id}" {if $prefs.fgal_root_id eq $galleryId}selected="selected"{/if} style="font-style:italic; border-bottom:1px dashed #666;">{tr}File Galleries{/tr}</option>
 				{section name=idx loop=$galleries}
 					{if $galleries[idx].id neq $prefs.fgal_root_id and $galleries[idx].perms.tiki_p_upload_files eq 'y'}
 					<option value="{$galleries[idx].id|escape}" {if $galleries[idx].id eq $galleryId}selected="selected"{/if}>{$galleries[idx].name|escape}</option>
@@ -289,12 +324,12 @@
 	<span class="attention">{tr}The file is locked by {$fileInfo.lockedby}{/tr}</span>
 {/if}
 <br />
-{remarksbox type="note"}
+{*{remarksbox type="note"}
 	{tr}Maximum file size is around:{/tr}
 	{if $tiki_p_admin eq 'y'}<a title="{$max_upload_size_comment}">{/if}
 		{$max_upload_size|kbsize:true:0}
 	{if $tiki_p_admin eq 'y'}</a>{/if}
-{/remarksbox}
+{/remarksbox}*}
 
 </div>
 
