--- tiki-adminusers.php	2010-11-22 10:10:24.000000000 -0200
+++ tiki-adminusers.php	2010-11-23 11:43:31.140285001 -0200
@@ -256,6 +256,10 @@
 				}
 				$cookietab = '1';
 				$_REQUEST['find'] = $_REQUEST['login'];
+
+				//codigo customizado para o pubeduc
+				$userlib->set_user_preference($_REQUEST['email'], 'realName', $_REQUEST['realName']);
+				$userlib->set_user_preference($_REQUEST['email'], 'other_information', $_REQUEST['other_information']);
 			} else {
 				$errors[] = array(
 					'num' => 1,
@@ -484,6 +488,28 @@
 	if (isset($_POST["edituser"]) and isset($_POST['login']) and isset($_POST['email'])) {
 		$access->check_authenticity(tra("Are you sure you want to modify this user's data?"));
 		if (!empty($_POST['login'])) {
+			
+			//codigo customizado para o pubeduc
+			if ($userinfo['realName'] != $_POST['realName']) {
+				$userlib->set_user_preference($_POST['login'], 'realName', $_POST['realName']);
+				$userinfo['realName'] = $_POST['realName'];
+				// HACK ALERT: na verdade o que precisa fazer eh soh mudar o valor do nome para o usuario no $users e nao pegar todos os usuarios de novo
+				$users = $userlib->get_users($offset, $numrows, $sort_mode, $find, $initial, true, $filterGroup, $filterEmail);
+				$smarty->assign_by_ref('users', $users["data"]);
+				$tikifeedback[] = array(
+					'num' => 0,
+					'mes' => 'Nome alterado com sucesso'
+				);
+			}
+			if ($userinfo['other_information'] != $_POST['other_information']) {
+				$userlib->set_user_preference($_POST['login'], 'other_information', $_POST['other_information']);
+				$userinfo['other_information'] = $_POST['other_information'];
+				$tikifeedback[] = array(
+					'num' => 0,
+					'mes' => 'Informações adicionais alteradas com sucesso'
+				);
+			}
+
 			if ($userinfo['login'] != $_POST['login'] && $userinfo['login'] != 'admin') {
 				if ($userlib->user_exists($_POST['login'])) {
 					$errors[] = array(
