--- templates/tiki-bot_bar.tpl	2010-09-09 10:01:37.720239123 -0300
+++ templates/tiki-bot_bar.tpl	2010-11-23 17:12:40.301046999 -0200
@@ -79,7 +79,7 @@
 		{tr}Powered by{/tr} <a href="http://tiki.org" title="&#169; 2002&#8211;{$smarty.now|date_format:"%Y"} {tr}The Tiki Community{/tr}">{tr}Tiki Wiki CMS Groupware{/tr}</a> {if $prefs.feature_topbar_version eq 'y'} v{$tiki_version} {if $tiki_uses_svn eq 'y'} (SVN){/if} -{$tiki_star}- {/if} | 
 	{/if}
 	<div id="credits">
-		{include file='credits.tpl'}
+	{*	{include file='credits.tpl'}*}
 	</div>
 </div>
 </div>
