--- templates/tiki-adminusers.tpl	2010-11-11 09:16:31.888828003 -0200
+++ templates/tiki-adminusers.tpl	2010-11-23 17:15:02.401046999 -0200
@@ -361,6 +361,14 @@
 						{/if}
 					</td>
 				</tr>
+
+				<tr class="formcolor">
+					<td><label for="realName">Nome:</label></td>
+					<td>
+						<input type="text" name="realName" id="realName" value="{$userinfo.realName}" />
+					</td>
+				</tr>
+
 				{*
 					No need to specify user password or to ask him to change it, if :
 					--> Tiki is using the Tiki + PEAR Auth systems
@@ -430,7 +438,14 @@
 						</tr>
 					{/if}
 				{/if}
-				
+
+				<tr class="formcolor">
+					<td><label for="other_information">Informações adicionais:</label></td>
+					<td>
+						<textarea rows="5" cols="33" name="other_information" id="other_information" />{$userinfo.other_information}</textarea>
+					</td>
+				</tr>
+			
 				{if $prefs.login_is_email neq 'y'}
 					<tr>
 						<td><label for="email">{tr}Email:{/tr}</label></td>
