--- lib/tikilib.php	2010-02-23 20:09:21.000000000 -0300
+++ lib/tikilib.php	2010-04-27 18:10:01.000000000 -0300
@@ -2186,6 +2186,8 @@
 				'tf.`lockedby`' => "'' as `lockedby`",
 				'tf.`comment`' => "'' as `comment`",
 				'tf.`archiveId`' => '0 as `archiveId`',
+				'tf.`subtitle`' => "'' as `subtitle`",
+				'tf.`credit`' => "'' as `credit`",
 				"'' as `visible`" => 'tfg.`visible`',
 				"'' as `public`" => 'tfg.`public`',
