--- ../tiki40/templates/tiki-upload_file.tpl	2010-03-19 01:42:19.000000000 -0300
+++ templates/tiki-upload_file.tpl	2010-03-23 19:22:49.000000000 -0300
@@ -17,9 +17,9 @@
 	{if count($galleries) > 0 and $tiki_p_list_file_galleries eq 'y'}
 		{if $filegals_manager neq ''}
 			{assign var=fgmanager value=$filegals_manager|escape}
-			{button href="tiki-list_file_gallery.php?filegals_manager=$fgmanager" _text="{tr}List Galleries{/tr}"}
+			{button href="$base_url" _text="{tr}List Galleries{/tr}"}
 		{else}
-			{button href="tiki-list_file_gallery.php" _text="{tr}List Galleries{/tr}"}
+			{button href="$base_url" _text="{tr}List Galleries{/tr}"}
 		{/if}
 	{/if}
 	{if count($uploads) > 0}
@@ -164,10 +164,9 @@
 				<input type="hidden" name="galleryId" value="{$prefs.fgal_root_id}"/>
 			{elseif empty($groupforalert)}
 			<tr><td>
-				<label for="galleryId">{tr}File gallery:{/tr}</label>
+				<label for="galleryId">Selecionar área:</label>
 			</td><td width="80%">
 				<select id="galleryId" name="galleryId[]">
-					<option value="{$prefs.fgal_root_id}" {if $prefs.fgal_root_id eq $galleryId}selected="selected"{/if} style="font-style:italic; border-bottom:1px dashed #666;">{tr}File Galleries{/tr}</option>
 				{section name=idx loop=$galleries}
 					{if $galleries[idx].id neq $prefs.fgal_root_id and
 						( ($galleries[idx].individual eq 'n') or ($galleries[idx].individual_tiki_p_upload_files eq 'y') )
@@ -284,12 +283,12 @@
 	<span class="attention">{tr}The file is locked by {$fileInfo.lockedby}{/tr}</span>
 {/if}
 <br />
-{remarksbox type="note"}
+{*{remarksbox type="note"}
 	{tr}Maximum file size is around:{/tr}
 	{if $tiki_p_admin eq 'y'}<a title="{$max_upload_size_comment}">{/if}
 		{$max_upload_size|kbsize:true:0}
 	{if $tiki_p_admin eq 'y'}</a>{/if}
-{/remarksbox}
+{/remarksbox}*}
 
 </div>
 
