--- lib/userslib.php	2010-11-17 10:36:12.000000000 -0200
+++ lib/userslib.php	2010-11-23 12:50:00.330285002 -0200
@@ -1358,6 +1358,10 @@
 			$res["age"] = $this->now - $res["registrationDate"];
 			$res['user_information'] = $this->get_user_preference($user, 'user_information', 'public');
 
+			// codigo adicionado para o sistema do pubeduc
+			$res['realName'] = $this->get_user_preference($user, 'realName', '');
+			$res['other_information'] = $this->get_user_preference($user, 'other_information', '');
+
 			$res['editable'] = $this->user_can_be_edited($user);
 		}
 
@@ -1921,6 +1925,10 @@
 
 			$res['editable'] = $this->user_can_be_edited($res['login']);
 
+			// codigo adicionado para o sistema do pubeduc
+			$res['realName'] = $this->get_user_preference($res['login'], 'realName', '');
+			$res['other_information'] = $this->get_user_preference($res['login'], 'other_information', '');
+
 			return $res;
 		}
 	}
